src/azul/logging.py

```diff
@@ -6,9 +6,39 @@
 )
 
 
+# lambda python bootstrap.py line 284 - to add format attribute of aws_request_id
+class LambdaLoggerFilter(logging.Filter):
+
+    def __init__(self, app: AzulChaliceApp = None):
+        try:
+            # From https://github.com/aws/aws-lambda-base-images/tree/python3.8
+            # Imports global variable present in lambda environment
+            from bootstrap import (  # noqa:AZUL141
+                _GLOBAL_AWS_REQUEST_ID,
+            )
+        except ImportError:
+            # Arbitrary UUID for local testing in place of non-existent lambda env var
+            self.aws_request_id = "06e9037a-97c8-4525-9b43-64c281eb06d2"
+        else:
+            assert app.lambda_context.aws_request_id == _GLOBAL_AWS_REQUEST_ID
+            self.aws_request_id = _GLOBAL_AWS_REQUEST_ID
+
+    def filter(self, record):
+        record.aws_request_id = self.aws_request_id
+        return True
+
+
 def configure_app_logging(app: AzulChaliceApp, *loggers):
     _configure_log_levels(app.log, *loggers)
 
+    rootlogger = logging.getLogger()
+    if len(rootlogger.handlers) != 0:
```

Comment:
When is the len(rootlogger.handler) == 0? 
It seems like something should be done within the else branch, should we set a handler for the rootlogger?